data-integration/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
+generated
```

Comment:
May have to remove since we need to track changes to the generated code.

Needs to be committed too.